docs/secure-connections/sauce-connect/setup-configuration/environment-variables.md

```diff
@@ -8,21 +8,118 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
-| `SAUCE_ARES_FLAGS` | Sets advanced flags for DNS library use with Sauce Connect Proxy. For more information, see the [c-ares Documentation](http://c-ares.haxx.se/ares_init.html). | Windows, Linux, Mac | n/a |
 
-For best performance, use the [latest version of Sauce Connect Proxy](/secure-connections/sauce-connect/installation).
+## Running Sauce Connect Proxy With Environment Variables
+
+  <Tabs
+      defaultValue="Mac/Linux"
+      values={[
+        {label: 'Mac/Linux', value: 'Mac/Linux'},
+        {label: 'Windows', value: 'Windows'},
+      ]}>
+
+<TabItem value="Mac/Linux">
+
+Follow the steps below to configure Sauce Connect Proxy using environment variables in the current terminal.
+
+1. In your terminal window, set the following environment variables
+  ```bash
+  export SAUCE_USERNAME="your Sauce username"
+  export SAUCE_ACCESS_KEY="your Sauce access key"
+  ```
+2. Starting a new Sauce Connect Proxy will not require adding `--api-key` or `--user` flags.
+  ```bash
+  sc --region us-west
+  ```
+
+Alternatively, Sauce Connect Proxy environment variables may be added to one of the user environment configuration files, such as `~/.bash_profile`.
+
+1. To make environment variables always available, open `~/.bash_profile` in your prefered text editor.
+2. Add the variables, for example:
+ ```bash
+ export SAUCE_USERNAME="your Sauce username"
+ export SAUCE_ACCESS_KEY="your Sauce access key"
+ ```
+3. Start a new shell or a new terminal.
+4. Confirm that your environment variables have been set by typing `echo $SAUCE_USERNAME` in your terminal. The response should be your username value.
+5. Just as above, starting a new Sauce Connect Proxy will not require adding `--api-key` or `--user` flags.
+ ```bash
+ sc --region us-west
+ ```
+
+</TabItem>
+<TabItem value="Windows">
+
+1. Open the Control Panel and click the System icon to open the **System Properties** dialog.
+2. Click **Environment Variables** to open the **Environment Variables** dialog.
+3. In the **User variables** section, click **New** to open the **New System Variable** dialog.
+4. For **Variable name**, enter **SAUCE_USERNAME** and for **Variable value**, enter your Sauce username and then click **OK**.
+5. Repeat 3-4 to set up the **SAUCE_ACCESS_KEY** or any other environment variable.
+6. Confirm that your environment variables have been set by typing `echo %SAUCE_USERNAME%` in your terminal. The response should be your username value. Then do the same for your access key.
+7. Starting a new Sauce Connect Proxy will not require adding `--api-key` or `--user` flags.
+ ```bash
+ sc --region us-west
+ ```
+
+</TabItem>
+</Tabs>
```

Comment:
@waggledans Most of these steps are covered in the [Setting Up Env Variables doc](https://docs.saucelabs.com/basics/environment-variables/). Our style is to do single sourcing: reuse existing written content instead of duplicating in multiple places (double the work to maintain!). 

Since these are in production already, I can edit down this section @sweeneyskirt-sl in a separate PR.